GoogleForCreators/web-stories-wp
# Animation: Circle Alignment on Bounce Animation part

## Summary
This fixes the alignment issue seen here in the fitness template while the dots where animating:

<img width="97" alt="093f3e0f-b2aa-459b-82d5-587dbb679a72" src="https://user-images.githubusercontent.com/35983235/90546878-eb9bb800-e147-11ea-9eea-61b563ef0974.png">

## Relevant Technical Choices
Turns out this is a chrome problem, I'm guessing it's an issue of chrome internally rounding off a needed level of precision when scaling clip paths that need super high precision like this one. You can see the issue highlighted here where I've applied a large scale transform to the circle element. The element itself is scaling properly, but the calculation for the clip-path is not:

<img width="959" alt="Screen Shot 2020-08-18 at 11 39 48 AM" src="https://user-images.githubusercontent.com/35983235/90547179-53ea9980-e148-11ea-8601-c1b68b271563.png">

Being an internal chrome problem, the only solution was finding a workaround. To accomplish this, I made a bullseye shape for the masks panel that combines the outer and inner circle. This solution wasn't affected by chrome's precision issues for scaling. It also provides the user a better experience regardless of animations cus they can select the single bullseye object and alter the color.

## To-do
NA

## User-facing changes
Bullseye now one shape in Fitness template.

## Testing Instructions
Go to Templates -> Fitness -> Create Story from template -> navigate to page 7 where these bullseyes appear & change the color of one. Color should change the entire bullseye (both dot and circle).

## Screenshots
**(Bouncing Bullseyes don't misalign)**
![bounce_alignment_bug](https://user-images.githubusercontent.com/35983235/90547718-2eaa5b00-e149-11ea-8652-61a0d2d431a8.gif)


---



Fixes #3115
